docs/reference/deploy/master-config-reference.rst

```diff
@@ -230,6 +230,13 @@ customize the UI.
 Specifies the path to a user-provided logo to be shown in the UI. Ensure the path is accessible and
 reachable by the master service. The logo file should be a valid image format, with SVG recommended.
 
+Logo path is defined in four variations:
+
+-  ``dark_horizontal``: The logo to be shown in the dark theme in the horizontal layout.
```

Comment:
Should we change "The logo to be shown ... " to "The path to the logo to be shown ..."? Or do you think it's clear that a path should be defined here? I feel a bit biased to leave things as is since I already know how to specify this in the config 😅

Reply:
I updated it